src/main/java/cloud/fogbow/common/constants/GoogleCloudConstants.java

```diff
@@ -1,11 +1,70 @@
 package cloud.fogbow.common.constants;
 
 public class GoogleCloudConstants {
+
+
     public static final String AUTHORIZATION_KEY = "Authorization";
     public static final String BEARER_S = "Bearer %s";
+    public static final String COMPUTE_V1_API_ENDPOINT = "/v1";
     public static final String ENDPOINT_SEPARATOR = "/";
+    public static final String VOLUME_ENDPOINT = "/volume";
+    public static final String COMPUTE_ENDPOINT = "/compute";
+    public static final String ZONES_KEY_ENDPOINT = "/zones";
+    public static final String INSTANCES_KEY_ENDPOINT = "/instances";
+    public static final String ATTACH_DISK_KEY_ENDPOINT = "/attachDisk";
+    public static final String DETACH_DISK_KEY_ENDPOINT = "/DetachDisk";
+    public static final String PREDEFINED_ZONE = "";
+    public static final String DEVICE_NAME_QUERY_PARAM = "?deviceName=";
+    public static final String DEFAULT_ZONE = "southamerica-east1-b";
+    public static final String DISKS_KEY_ENDPOINT = "/disks";
+    public static final String PROJECTS_KEY_ENDPOINT = "/projects";
+    public static final String PATH_PROJECT = "/projects";
+    public static final String PATH_ZONE = "/zones";
+    public static final String LINE_SEPARATOR = "/";
+    public static final String V1_API_ENDPOINT = "/v1";
+    public static final String GLOBAL_IP_ENDPOINT = "/global/addresses";
+    public static final String GLOBAL_IMAGES_ENDPOINT = "/global/images";
+    public static final String GLOBAL_NETWORKS_ENDPOINT = "/global/networks";
+    public static final String COMPUTE_ENGINE_ENDPOINT = "/compute";
+    public static final String COMPUTE_ENGINE_V1_ENDPOINT = "/v1";
+    public static final String FLAVOR_ENDPOINT = "/machineTypes";
+    public static final String COMPUTE_ATT_ENDPOINT = "/instances";
+    public static final String ZONE_KEY_CONFIG = "zone";
+    public static final String STATUS_KEY_JSON = "status";
+    public static final String GLOBAL_IP_NETWORK = "/global/networks";
+    public static final String REGION_ENDPOINT = "/regions";
+    public static final String SUBNET_ENDPOINT = "/subnetworks";
```

Comment:
There are multiple constants with the same value. There should be just one. For instance, both PROJECTS_KEY_ENDPOINT and PATH_PROJECT are set to "/projects". Please review all the above constants. Also, use a consistent naming convention.

Reply:
Okay, a new PR should be made with the proper modifications.